include/linux/soundwire/sdw_intel.h

```diff
@@ -365,7 +365,7 @@ struct sdw_intel_res {
  * on e.g. which machine driver to select (I2S mode, HDaudio or
  * SoundWire).
  */
-int sdw_intel_acpi_scan(acpi_handle *parent_handle,
```

Comment:
@ujfalusi can you please give a little more info on how you identified and fixed this?

Reply:
We have encountered a kernel crash on a developer setup. It might be caused by incorrect ACPI information. The kernel should never crash even if the ACPI information is incorrect.

Reply:
ok thanks!

Reply:
not clear why/how the code 'worked' since the beginning despite this mistake? Very odd...